WIP-CHANGELOG.md

```diff
@@ -1,44 +1,45 @@
-- `elasticsearch.storageClass` replaced by `elasticsearch.dataNode.storageClass`
-- `elasticsearch.retention.kubeAuditSize` replaced by `elasticsearch.retention.kubeAuditSizeGB`
-- `elasticsearch.retention.kubeAuditAge` replaced by `elasticsearch.retention.kubeAuditAgeDays`
-- `elasticsearch.retention.kubernetesSize` replaced by `elasticsearch.retention.kubernetesSizeGB`
-- `elasticsearch.retention.kubernetesAge` replaced by `elasticsearch.retention.kubernetesAgeDays`
-- `elasticsearch.retention.otherSize` replaced by `elasticsearch.retention.otherSizeGB`
-- `elasticsearch.retention.otherAge` replaced by `elasticsearch.retention.otherAgeDays`
+- Configuration changes requires running init again to get new default values.
+- Run the following migration script to update the object storage configuration: `migration/v0.7.x-v0.8.x/migrate-object-storage.sh`
+- The following configuration options must be manually updated:
+  - The config option `nginxIngress.controller.daemonset.useHostPort` has been replaced by `ingressNginx.controller.useHostPort`.
+    Make sure to remove the old option from your config when upgrading.
+  - Move `useRegionEndpoint` from `elasticsearch` to `fluentd` in `sc-config.yaml` before upgrading.
+  - The workload cluster config option `prometheus.retention.alertManager` has been removed.
+    Make sure to remove the option from your config when upgrading.
+  - The configuration for InfluxDB has been changed.
+    New default values will be added to your config, please update them to match your old values.
+    The following options has been removed or replaced
+    - `influxDB.address` removed
+    - `influxDB.metrics.sizeWc` replaced by `influxDB.retention.sizeWC`
+    - `influxDB.metrics.sizeSc` replaced by `influxDB.retention.sizeSC`
+    - `influxDB.retention.ageWc` replaced by `influxDB.retention.durationWC`
+    - `influxDB.retention.ageSc` replaced by `influxDB.retention.durationSC`
+  - The config for opendistro has been changed.
+    Please update the new defaults to match your old values.
+    The following options has been removed or replaced
+    - `elasticsearch.tolerations` removed
+    - `elasticsearch.nodeSelector` removed
+    - `elasticsearch.affinity` removed
+    - `elasticsearch.storageClass` replaced by `elasticsearch.dataNode.storageClass`
+    - `elasticsearch.retention.kubeAuditSize` replaced by `elasticsearch.curator.retention.kubeAuditSizeGB`
+    - `elasticsearch.retention.kubeAuditAge` replaced by `elasticsearch.curator.retention.kubeAuditAgeDays`
+    - `elasticsearch.retention.kubernetesSize` replaced by `elasticsearch.curator.retention.kubernetesSizeGB`
+    - `elasticsearch.retention.kubernetesAge` replaced by `elasticsearch.curator.retention.kubernetesAgeDays`
+    - `elasticsearch.retention.otherSize` replaced by `elasticsearch.curator.retention.otherSizeGB`
+    - `elasticsearch.retention.otherAge` replaced by `elasticsearch.curator.retention.otherAgeDays`
```

Comment:
Most if not all of these steps are included in the migration document. I think that's great but I think that also having it in the changelog is a bit redundant. I think you could change the whole `Release notes` to just include a reference to the migration document and a clear instruction that the operator must follow that document when upgrading.

Reply:
Good point! I'll "minimize" the Release notes :+1: